airbyte-integrations/connectors/source-recharge/source_recharge/manifest.yaml

```diff
@@ -0,0 +1,269 @@
+    record_selector:
+      $ref: "#/definitions/selector"
+    requester:
+      $ref: "#/definitions/requester_modern_api"
+    paginator:
+      $ref: "#/definitions/paginator_modern_api"
+    # we should ignore all other req.params once we have the `next_page_token` in response
+    # for pagination in `2021-11` - modern api.
+    ignore_stream_slicer_parameters_on_paginated_requests: true
+  # RETRIEVER FOR `METAFIELDS` STREAM
+  retriever_metafields:
+    $ref: "#/definitions/retriever_api_modern"
+    partition_router:
+      type: ListPartitionRouter
+      cursor_field: owner_resource
+      values:
+        - address
+        - order
+        - charge
+        - customer
+        - store
+        - subscription
+      request_option:
+        inject_into: request_parameter
+        type: RequestOption
+        field_name: owner_resource
+
+  # BASE STREAMS
+  # FULL-REFRESH
+  base_stream:
+    primary_key: "id"
+    schema_loader:
+      $ref: "#/definitions/schema_loader"
+  base_deprecated_api_stream:
+    $ref: "#/definitions/base_stream"
+    retriever:
+      $ref: "#/definitions/retriever_api_deprecated"
+  base_modern_api_stream:
+    $ref: "#/definitions/base_stream"
+    retriever:
+      $ref: "#/definitions/retriever_api_modern"
+  # INCREMENTAL
+  base_incremental_stream:
+    $ref: "#/definitions/base_modern_api_stream"
+    incremental_sync:
+      type: CustomIncrementalSync
+      class_name: source_recharge.components.datetime_based_cursor.RechargeDateTimeBasedCursor
+      cursor_field: "updated_at"
+      cursor_datetime_formats:
+        - "%Y-%m-%dT%H:%M:%S%z"
+      datetime_format: "%Y-%m-%dT%H:%M:%S%z"
+      start_datetime:
+        type: MinMaxDatetime
+        datetime: "{{ config['start_date'] }}"
+        datetime_format: "%Y-%m-%dT%H:%M:%SZ"
+      start_time_option:
+        type: RequestOption
+        field_name: "updated_at_min"
+        inject_into: request_parameter
+
+  # FULL-REFRESH STREAMS
+  # COLLECTIONS
+  collections_stream:
+    description: >-
+      Collections Stream: https://developer.rechargepayments.com/2021-11/collections/collections_list
+    $ref: "#/definitions/base_modern_api_stream"
+    $parameters:
+      name: "collections"
+      data_path: "collections"
+  # METAFIELDS
+  metafields_stream:
+    description: >-
+      Metafields Stream: https://developer.rechargepayments.com/2021-11/metafields
+    $ref: "#/definitions/base_modern_api_stream"
+    retriever:
+      $ref: "#/definitions/retriever_metafields"
+    $parameters:
+      name: "metafields"
+      data_path: "metafields"
+  # PRODUCTS
+  products_stream:
+    description: >-
+      Products Stream: https://developer.rechargepayments.com/2021-11/products/products_list
+      Products endpoint has 422 error with 2021-11 API version
+    $ref: "#/definitions/base_deprecated_api_stream"
+    $parameters:
+      name: "products"
+      data_path: "products"
+  # SHOP
+  shop_stream:
```

Comment:
Difference observed: 
* The low-code version sends the query param `limit=250` while this endpoint does not support pagination. Can we remove pagination for this stream?

Reply:
Good catch! Set the `Paginator` for the `shop` stream specifically to `NoPagination`